x-pack/plugins/fleet/public/components/uninstall_command_flyout/commands_for_platforms.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonGroup, EuiCodeBlock, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useMemo } from 'react';
+
+import type { PLATFORM_TYPE } from '../../hooks';
+import { PLATFORM_OPTIONS, usePlatform } from '../../hooks';
+
+import type { Commands } from './types';
+
+interface Props {
+  commands: Commands;
+}
+
+export const CommandsForPlatforms: React.FunctionComponent<Props> = ({ commands }) => {
+  const { platform, setPlatform } = usePlatform();
+
+  const options = useMemo(
+    () => PLATFORM_OPTIONS.filter(({ id }) => commands[id as PLATFORM_TYPE]),
+    [commands]
+  );
+
+  return (
+    <>
+      <EuiButtonGroup
+        options={options}
+        idSelected={platform}
+        onChange={(id) => setPlatform(id as PLATFORM_TYPE)}
+        legend={i18n.translate('xpack.fleet.agentUninstallCommand.platformSelectAriaLabel', {
```

Comment:
should this be: `xpack.fleet.agentUninstallCommandFlyout.platformSelectAriaLabel` ?